sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLListener.scala

```diff
@@ -46,6 +46,7 @@ case class SparkListenerSQLExecutionStart(
     description: String,
     details: String,
     physicalPlanDescription: String,
+    modifiedConfigs: Map[String, String],
```

Comment:
Adding a field here will break the backward compatibility. The history server can't load the previous event logs.

Reply:
That's a good point, seems a better approach is add an another event.

Reply:
Can we resolve it by moving the new field to the tail of the class constructor?

Reply:
yeah, revert the new event which seems quite hack